restaurant-showcase/payment/src/main/java/restaurant/showcase/payment/PaymentController.java

```diff
@@ -0,0 +1,26 @@
+package restaurant.showcase.payment;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@Slf4j
+@RestController
+public class PaymentController {
+
+    private final static long ONE_SECOND = 1000L;
+
+    @RequestMapping(value = "/payment", method = RequestMethod.POST)
+    public String payment(@RequestBody OrderRO orderRO) throws InterruptedException {
+        for (int i = 0; i < 5; i++) {
```

Comment:
Magic number 5

Reply:
yes. it's random, since it's just logging.. would let it as is